Frameworks/CoreLocation/CLLocation.mm

```diff
@@ -228,11 +223,24 @@ - (id)copyWithZone:(NSZone*)zone {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
+ @Notes  This uses the "haversine" formula to find circular distance between two points, found
+         here: https://en.wikipedia.org/wiki/Haversine_formula
 */
 - (CLLocationDistance)distanceFromLocation:(const CLLocation*)location {
```

Comment:
Check when location is NULL. What is the expected behaviour when location is NULL?